test/routes/api/tasks/tasks.test.ts

```diff
@@ -622,6 +647,34 @@ describe('Tasks api (logged user only)', () => {
     })
 
     describe('Retrieval', () => {
+      before(async () => {
+        app = await build()
+
+        await app.knex<Task>('tasks').where({ id: taskId }).update({ filename: null })
+
+        const files = fs.readdirSync(uploadDirTask)
+        files.forEach((file) => {
+          const filePath = path.join(uploadDirTask, file)
+          fs.rmSync(filePath, { recursive: true })
+        })
```

Comment:
You can create a function `removeFiles`, but the tests are run sequentially for db consistency, if call it in after hook, they will be removed before you reach this test suite.

Reply:
> but the tests are run sequentially for db consistency, if called in after hook, they will be removed before you reach this test suite.

i didn't understand this

Reply:
I mean that tests are not run in parralel, if you remove the files you uploaded during a test suite in `after` hook, it hsoudl be enough. You should not have to remove the files in `before` hook.